filebeat/module/postgresql/log/_meta/fields.yml

```
@@ -8,17 +8,32 @@
       description: >
         The timestamp from the log line.
     - name: core_id
+      type: alias
+      path: session_line_number
```

Full path is needed here.
```suggestion
      path: postgresql.log.session_line_number
```